Makefile

```
@@ -0,0 +1,32 @@
+# Variables
+BINARY_NAME=thunder
+OUTPUT_DIR=target
+ZIP_FILE=product-is-thunder.zip
```

Do we need to have product-is references with this repo?

```suggestion
ZIP_FILE=thunder.zip
```

---

Already fixed with https://github.com/ThaminduDilshan/thunder/pull/1